pkg/controllers/v1alpha1/dataload/implement.go

```diff
@@ -197,7 +228,60 @@ func (r *DataLoadReconcilerImplement) reconcilePendingDataLoad(ctx reconcileRequ
 		return utils.RequeueAfterInterval(20 * time.Second)
 	}
 
-	// 5. lock the target dataset. Make sure only one DataLoad can win the lock and
+	// 5. Check existence of the targetPath in alluxio
+	var notExisted bool
+	switch boundedRuntime.Type {
+	case common.ALLUXIO_RUNTIME:
+		podName := fmt.Sprintf("%s-master-0", targetDataset.Name)
+		containerName := "alluxio-master"
+		fileUtils := operations.NewAlluxioFileUtils(podName, containerName, targetDataset.Namespace, ctx.Log)
+		for _, target := range ctx.DataLoad.Spec.Target {
+			isexist, err := fileUtils.IsExist(target.Path)
+			if err != nil {
+				return utils.RequeueAfterInterval(20 * time.Second)
+			}
+			if !isexist {
+				notExisted = true
+			}
+		}
+	default:
+		log.Error(fmt.Errorf("RuntimeNotSupported"), "The runtime is not supported yet", "runtime", boundedRuntime)
+		r.Recorder.Eventf(&ctx.DataLoad,
+			v1.EventTypeNormal,
+			common.RuntimeNotReady,
+			"Bounded accelerate runtime not supported")
+	}
+
+	if notExisted {
+		r.Recorder.Eventf(&ctx.DataLoad,
+			v1.EventTypeNormal,
```

Comment:
I think it better to change `EventTypeNormal` to `EventTypeWarning` since it will make DataLoad fail fast.

Reply:
tks, done.